packages/eslint-config-leboncoin/index.js

```diff
@@ -11,231 +11,250 @@ module.exports = {
+    "jsx-a11y/no-noninteractive-tabindex": "warn",
+    "jsx-a11y/no-autofocus": "warn",
+    "jsx-a11y/accessible-emoji": "warn",
+    "jsx-a11y/aria-role": "warn",
+    "jsx-a11y/iframe-has-title": "warn",
+    "jsx-a11y/anchor-has-content": "warn",
+    "@typescript-eslint/ban-ts-comment": "warn",
+    "@typescript-eslint/ban-types": "warn",
     "@typescript-eslint/explicit-function-return-type": "off",
-    '@typescript-eslint/no-var-requires': 'off',
-    '@typescript-eslint/no-empty-function': 'off',
-    '@typescript-eslint/no-use-before-define': 'off',
-    '@typescript-eslint/no-non-null-asserted-optional-chain': 'warn',
-    '@typescript-eslint/no-explicit-any': ["warn", { "ignoreRestArgs": true }],
+    "@typescript-eslint/no-var-requires": "off",
+    "@typescript-eslint/no-empty-function": "off",
+    "@typescript-eslint/no-use-before-define": "off",
+    "@typescript-eslint/no-non-null-asserted-optional-chain": "warn",
+    "@typescript-eslint/no-explicit-any": ["warn", { ignoreRestArgs: true }],
     "no-unused-expressions": "off",
     "no-unused-vars": "off",
-    "@typescript-eslint/no-unused-expressions": ["error", { "allowTernary": true, "allowShortCircuit": true }],
-    "@typescript-eslint/no-unused-vars": ["error", {"args": "after-used", "ignoreRestSiblings": true, "varsIgnorePattern": "[tT]ype" }],
-    "@typescript-eslint/explicit-module-boundary-types": 'off',
-    "array-callback-return": 'off',
-    "no-restricted-properties": ["error", {
-      "property": "whyDidYouRender"
-    }]
+    "@typescript-eslint/no-unused-expressions": [
+      "error",
+      { allowTernary: true, allowShortCircuit: true },
+    ],
+    "@typescript-eslint/no-unused-vars": [
+      "error",
+      {
+        args: "after-used",
+        ignoreRestSiblings: true,
+        varsIgnorePattern: "[tT]ype",
+      },
+    ],
+    "@typescript-eslint/explicit-module-boundary-types": "off",
+    "array-callback-return": "off",
+    "no-restricted-properties": [
+      "error",
+      {
+        property: "whyDidYouRender",
+      },
+    ],
   },
-  "overrides": [
+  overrides: [
     {
-      "files": ["*.ts", "*.tsx"],
-      "rules": {
+      files: ["*.ts", "*.tsx"],
+      rules: {
         "@typescript-eslint/explicit-function-return-type": ["warn"],
-        "react/prop-types": ["off"]
-      }
+        "react/prop-types": ["off"],
+      },
+    },
+    {
+      files: ["*.d.ts"],
+      rules: {
+        "@typescript-eslint/no-unused-vars": ["off"],
+      },
     },
     {
-      "files": ["*.d.ts"],
-      "rules": {
-        "@typescript-eslint/no-unused-vars": ['off']
-      }
+      files: ["*.spec.*"],
+      rules: {
+        "no-import-assign": ["off"],
+      },
     },
     {
-      "files": ["*.spec.*"],
-      "rules": {
-        "no-import-assign": ['off']
-      }
-    }
-  ]
-}
+      files: ["**/__tests__/**/*.[jt]s?(x)", "**/?(*.)+(spec|test).[jt]s?(x)"],
+      extends: ["plugin:testing-library/react"],
```

Comment:
This is the part that should be reviewed.

Reply:
regex hell :scream_cat: